code/modules/food_and_drinks/food/foods/pizza.dm

```diff
@@ -80,15 +80,15 @@
 	filling_color = "#e5b437"
 
 /obj/item/reagent_containers/food/snacks/sliceable/pizza/macpizza
-	name = "Macaroni cheese pizza"
+	name = "macaroni cheese pizza"
```

Comment:
After confirming with maintainers, we want to use the American spellings whenever possible. This should be one of the following:
- macaroni and cheese
- macaroni & cheese
- mac'n'cheese